pyproject.toml

```diff
@@ -0,0 +1,72 @@
+[build-system]
+requires = ["setuptools>=61.0", "setuptools_scm[toml]>=6.2"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools_scm]
+
+[project]
+name = "fastpair"
+dynamic = ["version"]
+authors = [
+    { name = "Carson J. Q. Farmer", email = "carsonfarmer@gmail.com" },
+]
+
+#maintainers = [{ name = "" }]
+
+license = { text = "MIT" }
+description = "FastPair: Data-structure for the dynamic closest-pair problem."
+keywords = ["closest-pair points algorithm fastpair"]
+readme = "README.md"
+classifiers = [
+    "Development Status :: 2 - Pre-Alpha",
+    "Environment :: Console",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Information Technology",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: MIT License",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Topic :: Scientific/Engineering :: Information Analysis",
+    "Topic :: System :: Distributed Computing",
+]
+
+requires-python = ">=3.10"
```

Comment:
is this going to be a problem, seems like a high-bar requirement?

Reply:
While we could support Python <3.10 [the SPEC000 support window](https://scientific-python.org/specs/spec-0000/#support-window) is actually closing on Python 3.10 pretty soon. Of course, we don't have to follow this explicitly, but I can say that it has been a life-saver in terms of clear cut minimum requirements.

If you would feel more comfortable in supporting an older minimum version, we can do that, but I wouldn't recommend anything older than Python 3.9.

It's also worth noting that SciPy itself follows SPEC000 and [requires `python>=3.10`](https://github.com/scipy/scipy/blob/3535b3780669d5b742813638461bca79d3358483/pyproject.toml#L47)

Reply:
Naw, all good. Was more just a question.